litmus-portal/frontend/src/components/YamlEditor/styles.ts

```diff
@@ -40,87 +58,268 @@ const useStyles = makeStyles((theme: Theme) => ({
 
   editorGrid: {
     overflow: 'auto',
-    maxHeight: '34.5rem',
-    width: '92%',
+    height: '50vh',
+    width: '100%',
+    '&::-webkit-scrollbar': {
+      width: '0.2em',
+    },
+    '&::-webkit-scrollbar-track': {
+      webkitBoxShadow: 'inset 0 0 6px rgba(0,0,0,0.00)',
+    },
+    '&::-webkit-scrollbar-thumb': {
+      backgroundColor: theme.palette.secondary.dark,
+      outline: '1px solid slategrey',
+    },
+    [theme.breakpoints.down('xl')]: {
+      height: '84vh',
+    },
+    [theme.breakpoints.down('lg')]: {
+      height: '56vh',
+    },
+    [theme.breakpoints.down('md')]: {
+      height: '49vh',
+    },
+
+    [theme.breakpoints.down('sm')]: {
+      height: '40vh',
+    },
+    [theme.breakpoints.down('xs')]: {
+      height: '30vh',
+    },
   },
 
   extraSpace: {
-    backgroundColor: '#161616',
+    backgroundColor: theme.palette.editorBackground,
     height: '2rem',
     width: '100%',
   },
 
   editorButtons: {
     borderRadius: 3,
-    backgroundColor: '#161616',
+    backgroundColor: theme.palette.editorBackground,
+    boxSizing: 'border-box',
+    color: theme.palette.secondary.contrastText,
+    borderColor: theme.palette.customColors.white(0.2),
+    paddingLeft: theme.spacing(3.125),
+    width: '4rem',
+    height: '2.75rem',
+    marginLeft: theme.spacing(1.25),
+  },
+
+  editorButtonGoto: {
+    borderRadius: 3,
+    backgroundColor: theme.palette.editorBackground,
+    boxSizing: 'border-box',
+    color: theme.palette.secondary.contrastText,
+    borderColor: theme.palette.customColors.white(0.2),
+    paddingLeft: theme.spacing(3.125),
+    width: '4rem',
+    height: '2.75rem',
+    marginLeft: theme.spacing(1.25),
+    [theme.breakpoints.down('xs')]: {
+      marginLeft: 0,
+      marginTop: theme.spacing(1.5),
+    },
+  },
+
+  editorButtonCopy: {
+    borderRadius: 3,
+    backgroundColor: theme.palette.editorBackground,
+    boxSizing: 'border-box',
+    color: theme.palette.secondary.contrastText,
```

Comment:
I can see a lot of common styles between the editor buttons...
this common css properties can be extracted into a single class and you can apply multiple class names in the HTML part using javascript template literals
like this
```
className={`${editorButton} ${editorButtonCopy}`}
```